feat: ja translate for getting-started

riywo opened the pull request:
Close #190

riywo committed:
feat: ja translate for getting-started

riywo commented:
@poppoerika Check this out!

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/getting-started.md:
```diff
@@ -1,68 +1,68 @@
+# Momento Cache と Momento Topics をはじめてみる
 
-If you're looking to jump in and get started with Momento Cache with an SDK, you've come to the right place.
+もし Momento Cache を SDK から使ってみる方法をお探しでしたら、ここが正解です。
```
`ここが正解です`　→ `ここはあなたにうってつけの場所です。`？

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/getting-started.md:
```diff
@@ -1,68 +1,68 @@
+## ステップ 2: Momento コンソールで認証トークンを作成する
 
-In the console, open the menu and select Token.
+コンソール上で、メニューを開き Token を選択します。
```
I'm adding localization to our console, so this can be `Token` -> `トークン`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/getting-started.md:
```diff
@@ -1,68 +1,68 @@
 ![image of console menu](/img/getting-started/auth-token.gif)
 
-On the token page, select your 1/ cloud provider, 2/ an available Region from the drop down list, 3/ when the token should expire, and then 4/ hit the "Generate Token" button.
+Token ページ上で、1. クラウドプロバイダーを選択し、2. ドロップダウンリストから利用可能なリージョンを選び、3. トークンの有効期限を設定し、そして 4. "Generate Token" ボタンをクリックします。
```
Same for here. `Generate Token` -> `トークンを生成する`

riywo commented:
@poppoerika Addressed your comment! https://github.com/momentohq/public-dev-docs/compare/2abe0aeb7d5546a3836d8a7a43de725f41ef8556..153f7ea21800189788e4ea5af8c6d658abd0d6e8
